### PR TITLE
Some modifications merged from Creator

### PR DESCRIPTION
**Pull request by pandamicro**

### CODE REVIEW

File: cocos/storage/local-storage/LocalStorage-android.cpp [hunk 1]
```diff
@@ -94,6 +94,10 @@ bool localStorageGetItem( const std::string& key, std::string *outItem )
     {
         jstring jkey = t.env->NewStringUTF(key.c_str());
         jstring jret = (jstring)t.env->CallStaticObjectMethod(t.classID, t.methodID, jkey);
+        if (jret == nullptr)
+        {
+            return false;
+        }
```
Comment (minggo) [inline cocos/storage/local-storage/LocalStorage-android.cpp:103]: I think it needs to delete local reference as the following codes do

``` c++
         t.env->DeleteLocalRef(jret);
          t.env->DeleteLocalRef(jkey);  
         t.env->DeleteLocalRef(t.classID);
```

File: cocos/platform/CCImage.h [hunk 2]
```diff
@@ -144,7 +144,13 @@ class CC_DLL Image : public Ref
...
      */
     bool saveToFile(const std::string &filename, bool isToRGB = true);
-    
+
+    /**
+     * Enables or disables premultiplied alpha for PNG files.
+     *
+     *  @param enabled (default: true)
+     */
+    static void setPNGPremultipliedAlphaEnabled(bool enabled) { PNG_PREMULTIPLIED_ALPHA_ENABLED = enabled; }
```
Comment (minggo) [inline cocos/platform/CCImage.h]: Please add static function before member functions as the rules in [coding styles](https://github.com/cocos2d/cocos2d-x/blob/v3/docs/CODING_STYLE.md#declaration-order).

Comment (pandamicro) [inline cocos/platform/CCImage.h]: Thanks for the remind

**Comment by pandamicro** [comment]
@minggo Fixed

**Pull request merged**